src/components/SignUpModal/component/ValidationInputs/ValidationInputs.tsx

```diff
@@ -94,11 +95,12 @@ const ValidatedInputs: React.FC<Props> = (props: Props) => {
     setParameter(event.target.value);
   };
 
-  const inputBlurHandler = (event: React.FocusEvent<HTMLInputElement>) => {
+  const generalInputHandler = () => {
+    const capitalizedName = name.charAt(0).toUpperCase() + name.slice(1);
     const alarmMes =
       // eslint-disable-next-line no-nested-ternary
       inputValue === ''
-        ? `${event.target.name} required`
+        ? `${capitalizedName} required`
         : !inputValue.match(regularExpression)
         ? formatPrompt
         : '';
```

Comment:
readable ?    and please dont disable eslint rule next time

Reply:
this is not written by me. I only modify some variable with his/her code. I am struggling watch the code, too. haha